test/extensions/quic_listeners/quiche/platform/quic_platform_test.cc

```diff
@@ -357,6 +357,19 @@ TEST(QuicPlatformTest, QuicDLog) {
 
 #undef VALUE_BY_COMPILE_MODE
 
+TEST(QuicPlatformTest, QuicCHECK) {
+  CHECK(1 == 1);
+  CHECK(1 == 1) << " 1 == 1 is forever true.";
+  EXPECT_DEBUG_DEATH({ DCHECK(false) << " Supposed to fail in debug mode."; },
```

Comment:
nit, feel free to ignore:  It'd be nicer visually if you

- Put the first 2 CHECKs in adjacent lines
- Then followed by an empty line
- Then put the 2 EXPECT_DEBUG_DEATHs in adjacent lines
- Then followed by an empty line
- Then put the 2 EXPECT_DEATHs in adjacent lines

Reply:
done